mcmc/mcmc.cpp

```diff
@@ -32,7 +32,8 @@ void mcmc::init(std::string outfile) {
   // Counter of the accepted # of steps
   accCount = 0;
   //KS: you don't want to do this too often https://root.cern/root/html606/TTree_8cxx_source.html#l01229
-  auto_save = fitMan->raw()["General.AutoSave"].as<int>();//GetAutoSave();
+  //auto_save = fitMan->raw()["General.AutoSave"].as<int>();//GetAutoSave();
```

Comment:
let's have this be configurable. I.e. let's read this in from fitMan->raw()["General"]["AutoSave"].as<int>

Reply:
Yep can do, can't remember why I did this anymore but can revert it